insights/specs/datasources/user_group.py

```diff
@@ -0,0 +1,24 @@
+"""
+Custom datasources for user and group related specs
+
+.. note::
+    It should be noted that the specs in this module might contains sensitive
+    information, please avoid collecting any of them.
+"""
+from insights.specs import Specs
+from insights.core.context import HostContext
+from insights.core.dr import SkipComponent
+from insights.core.plugins import datasource
+from insights.core.filters import get_filters
+
+
+@datasource(HostContext)
+def groups(broker):
```

Comment:
@xiangce this confused me at first.  Your not really collecting anything here, but just returning the list of filters for this data spec.  What do you think about renaming it to something like `group_filters` instead of `groups`?  Its an unexpected way of using filters, but I believe it provides the same net results.

Reply:
@bfahr - Thanks for the suggestion. I modified it to return the `list` directly instead of a string, and rename it as `group_filter_list`. What do you think?

Reply:
@bfahr - Updated. Thanks for the suggestion.

Reply:
@xiangce that looks good, thanks!